ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -28,7 +28,7 @@ function createMarkdownPanelStyles(theme: Theme, chartsTheme: PersesChartsTheme)
     height: '100%',
     overflowY: 'auto',
     // Ignore top margin on the first element.
-    '& :first-child': {
+    '& :first-of-type': {
```

Comment:
I got a console error when opening a dashboard:
```
The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".
```
We don't use SSSR, so it's a little bit strange... I need to check if it's the right solution or if we can't delete this property.

Reply:
We've seen a similar error with InfoTooltip before too, I'll try to find that fix

Reply:
Ah nvm, this is [the PR](https://github.com/perses/perses/pull/1068/files#r1156577532) I was thinking of

Hopefully what you did fixes it 🤞, always nice to reduce some console noise

Reply:
I did some check, the fix is working fine 😄